src/SFA.DAS.AssessorService.Data/StandardRepository.cs

```diff
@@ -249,7 +272,22 @@ private async Task<Standard> GetStandardsByStandardReferenceAndVersionInternal(s
 
             var results = await _unitOfWork.Connection.QueryAsync<Standard>(
                 sql,
-                param: new { standardReference, version },
+                param: new { ifateReferenceNumber, version },
+                transaction: _unitOfWork.Transaction);
+
+            return results.FirstOrDefault();
+        }
+
+        private async Task<Standard> GetStandardByLarsCodeAndVersionInternal(int larsCode, string version)
+        {
+            var sql = @"SELECT StandardUId, IfateReferenceNumber, LarsCode, Title, Version, Level, Status, TypicalDuration, 
+                            MaxFunding, IsActive, LastDateStarts, EffectiveTo, EffectiveFrom, VersionEarliestStartDate, 
```

Comment:
Column names are in square brackets on some queries and not others

Reply:
Have updated to keep it consistent